test/unit/specs/components/staking/ModalStake.spec.js

```diff
@@ -16,6 +17,10 @@ const Wrapper = () => {
   })
 }
 
+test(`renders correctly`, () => {
+  expect(htmlBeautify(Wrapper().html())).toMatchSnapshot()
```

Comment:
note: wrapper.vm.$el also prints a formated dom representation

Reply:
Thank you, how is that helpful?  Is it already beautified?

Reply:
Yes in a similar way.